### PR TITLE
Fix/occupancy

### LINKED ISSUE #26: URL for specific place doesnt work with national varieties
Seems Airbnb launched national version of the web (.cz, .fr ...). The url search needs readjusting.
https://my.apify.com/view/runs/olX1gHoqW1a4MzUJ9

### PR DESCRIPTION
**Pull request by lhotanok**
Fixes #26 
Add support for localized `startUrls` (such as https://www.airbnb.cz/rooms/37288141)

### CODE REVIEW

File: src/tools.js [hunk 1]
```diff
@@ -1,18 +1,179 @@
...
+        const requestUrl = new URL(url);
+        requestUrl.searchParams.set('locale', locale);
+
+        const options = {
+            url: requestUrl.toString(),
+            json: false,
+            headers: {
+                'X-Airbnb-API-Key': process.env.API_KEY,
+            },
+            proxyUrl: proxy.newUrl(session ? session.id : `airbnb_${Math.floor(Math.random() * 100000000)}`),
```
Comment (metalwarrior665) [inline src/tools.js:40]: Why random session number? We can just put there underfined

**Review by metalwarrior665 — COMMENTED** [review]
I'm thinking if there is a way to make these PRs smaller? E.g. first do only refactor and then only fix or vice versa? I know that they go hand in hand and it is hard to fix without refactor or refactor broken code. But at some point we will need to make the review process smoother. Let's think about it :)

Otherwise, let's merge and do after review maybe by @pocesar :)

**Comment by lhotanok** [comment]
> I'm thinking if there is a way to make these PRs smaller? E.g. first do only refactor and then only fix or vice versa? I know that they go hand in hand and it is hard to fix without refactor or refactor broken code. But at some point we will need to make the review process smoother. Let's think about it :)
> 
> Otherwise, let's merge and do after review maybe by @pocesar :)

@metalwarrior665 Yeah, I've been thinking about it as well 😅 This way I often get comments on code I didn't write myself and that was just moved to a different function. So for bigger issues, I'll try 2 PRs approach if refactoring is needed. Refactoring-first might work better, I don't like to add more code to the existing long code chain where you can barely follow variable scopes 😄

There could be potential issues with merge conflicts though. To make 2 PRs at the same moment, you need to have 2 different branches, right? So we would need to synchronize the PRs, merge them in the right order and if there was some other PR merged in the meantime, merge conflicts would have to be resolved in both branches.

File: src/tools.js (hunk 1 above)
Comment (lhotanok) [reply src/tools.js:40]: Didn't write this one but good to know 👍

**Pull request merged**